public/kibana-integrations/kibana-vis.js

```diff
@@ -438,7 +437,7 @@ class KibanaVis extends Component {
               <EuiIcon type="iInCircle" />
             </EuiToolTip>
           </div>
-          <div id={this.visID} vis-id={this.visID} style={{ height }}></div>
+          <div id={this.visID} vis-id={this.visID} style={{ display: this.props.resultState === 'loading' ? 'none' : 'block', height:"100%" }}></div>
```

Comment:
May be the code readability could be improved by moving the logic in the style outside the render